lib/event-listeners/foods-event-listeners.js

```diff
@@ -11,7 +11,7 @@ const newFoodSubmit = () => {
     let name = $('#new-food-form [name=new-food-name').val()
     let calories = $('#new-food-form [name=new-food-calories').val()
 
-    if (name === "") return missingFoodField(name)
+    if (name === "" || calories === "") return missingFoodField(name, calories)
 
     postNewFood(name, calories)
     renderNewFood(name, calories)
```

Comment:
This PR looks good - I think we should move this function to be called from the postNewFood fetch call, however - but otherwise GTG